src/database/seeds/02_Participants.ts

```diff
@@ -0,0 +1,39 @@
+import { Knex } from 'knex';
```

Comment:
I'm not sure ordering seed files is a good idea - if a seed file depends on data in another, imagine the order of operations where Users is first:

1. First run starts
2. Insert user A
3. Insert participant Z which adds user A to the team
4. Second run
5. Seed tries to delete and reinsert user A
6. Error!

I don't think we should rely on ordering at this stage - and Knex creates seeds without a leading number, so this breaks the pattern. If there's a good reason, sure.

Reply:
Maybe that's a bad example because the relationship is the other way - but I think we should solve this when it's a problem, not now.